src/blocks/media-block/image-block/SimpleImageBlock.test.tsx

```
@@ -0,0 +1,342 @@
+}
+
+// Create a mock image instance factory
+const createMockImageInstance = (width?: number, height?: number): HTMLImageElement => {
+  const img = document.createElement('img');
+  if (width !== undefined) img.width = width;
+  if (height !== undefined) img.height = height;
+  return img;
+};
+
+// Create a mock Image constructor function
+const MockImage = (function(this: HTMLImageElement | void, width?: number, height?: number): HTMLImageElement {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)